templates/zoo.cfg.j2

```diff
@@ -66,3 +66,7 @@ metricsProvider.className={{ zookeeper_metricsprovider_classname }}
 metricsProvider.httpPort={{ zookeeper_metricsprovider_httpport }}
 metricsProvider.exportJvmInfo={{ zookeeper_metricsprovider_exportjvminfo | ternary('true', 'false') }}
 {% endif %}
+
+{% for key, value in (zookeeper_cfg_params | default({})).items() %}
```

Comment:
Can you change the variable name to `zookeeper_config_params`.

Reply:
done 👍